src/internal/connector/sharepoint/api/pages.go

```diff
@@ -81,6 +83,22 @@ func GetSitePages(
 	return col, nil
 }
 
+func GetSiteLite(ctx context.Context, gs graph.Servicer, siteID string) (msmodels.Siteable, error) {
```

Comment:
Nit: since I don't see a `GetSite` otherwise.
```suggestion
func GetSite(ctx context.Context, gs graph.Servicer, siteID string) (msmodels.Siteable, error) {
```

Reply:
Resolved with [ce8c12d](https://github.com/alcionai/corso/pull/2461/commits/ce8c12d20e576eacd5152a5ac629305c52d6eff1)